src/analyzer.ts

```diff
@@ -261,7 +273,7 @@ export class Analyzer {
       warnings.push({
         code: 'could-not-load',
         message: `Unable to load import: ${error.message || error}`,
-        sourceRange: scannedImport.sourceRange,
+        sourceRange: scannedImport.urlSourceRange || scannedImport.sourceRange,
```

Comment:
If we want to point out a problem with the url, like so:

```
<link rel="import" href="./nonexistant.html">
                        ^------------------^
```

If we don't know where the URL is, it's at least a reasonable fallback to do:

```
<link rel="import" href="./nonexistant.html">
^-------------------------------------------^
```